src/Raven.Client/Documents/Operations/TimeSeries/GetTimeSeriesOperation.cs

```diff
@@ -77,9 +77,10 @@ public override HttpRequestMessage CreateRequest(JsonOperationContext ctx, Serve
                 pathBuilder.Append("&name=")
                     .Append(Uri.EscapeDataString(_name))
                     .Append("&from=")
-                    .Append(_from.EnsureUtc().GetDefaultRavenFormat())
+                    .Append(_from?.EnsureUtc().GetDefaultRavenFormat())
```

Comment:
it will be better if we will not provide `from` in the URL at all

Reply:
The thing is that we expect pairs of (from, to). 
For getting multiple time-series there might be a problem if the first one will pass only `from` and the second one only `to`..

Reply:
For ranges, I agree. But in this method?